modules/elprep/filter/main.nf

```diff
@@ -65,7 +65,7 @@ process ELPREP_FILTER {
     def assembly_regions_cmd = get_assembly_regions ? " --assembly-regions ${prefix}.assembly_regions.igv": ""
 
     """
-    elprep filter ${bam} ${prefix}.${suffix} \\
+    elprep filter ${bam} output/${prefix}.${suffix} \\
```

Comment:
Where is `output/` defined here?

Reply:
nowhere, I needed to define a random output directory name to avoid the collision with the input bam

Reply:
so the bam file is written to `output/test.bam`

Reply:
Sorry completely missed in input ${bam} because I'm too used to one line per param/argument :facepalm:

Reply:
No problem, perhaps I should split it up into lines next time 😄